pkg/restore/restore_new_test.go

```diff
@@ -1749,7 +1749,7 @@ func TestRestorePersistentVolumes(t *testing.T) {
 				).
 				addItems("persistentvolumeclaims",
 					test.NewPVC("ns-1", "pvc-1",
-						test.WithVolumeName("pv-1"),
+						test.WithPVName("pv-1"),
```

Comment:
This function was renamed to be clearer from a Velero perspective. A `Volume` != a `PersistentVolume`, but we're working with `PersistentVolume`s here.

Reply:
WFM

Reply:
Just need to switch to making this change in `restore_test.go`, since `restore_new_test.go` has been renamed.